File: Films.java
import java.util.*;

public class Films {
    public static void main(String[] args) {
        //list films
        List<String> Films = new ArrayList<>();
        Films.add("Avatar");
        Films.add("Dumb and Dumber");
        Films.add("Titanic");
        Films.add("Pretty Woman");
        Films.add("The Grinch");
        Films.add("Man in black");

        // remove film "Titanic"
        Films.remove("Titanic");

        // add film " The Wolf of Wall Street"
        Films.add("The Wolf of Wall Street");

        // length of list films
        System.out.println(Films.size());

        // the index of "The Grinch" film
        System.out.println(Films.indexOf("The Grinch"));

        if (!Films.isEmpty()) {
            System.out.println("We don't go to cinema");
        }    else {
            System.out.println("We go to cinema");

        }

        // list the elements
        System.out.println(Films);

        // duration of films
        List<Integer> durationFilms = Arrays.asList(new Integer[]{180, 101, 90, 100, 96, 170});
        System.out.println(durationFilms);

        // category of films
        List<String> categoryFilms = new ArrayList<>();
        categoryFilms.add("fantasy");
        categoryFilms.add("comedy");
        categoryFilms.add("romantic");
        categoryFilms.add("animation");
        categoryFilms.add("scifi");
        categoryFilms.add("drama");

        System.out.println(categoryFilms);

        // films and duration
        Map<String,Integer> films = new HashMap<String,Integer>()
        {
            {
                put("Avatar", 180);
                put("Dumb and Dumber", 101);
                put("Pretty Woman", 90);
                put("The Grinch", 100);
                put("Man in black", 96);
                put("The Wolf of Wall Street", 170);


            }};
        System.out.println(films);

        // good or bad film
        boolean good_film = false;
        if (good_film = true);
        {

            System.out.println("We will not watch");
            System.out.println("We will watch");
        }

    }
}

File: films.py
films = ['Avatar', 'Dumb and Dumber', 'Titanic','Pretty Woman', 'The Grinch','Man in Black']

print(films)

# remove film 'Titanic
films.remove('Titanic')
print(films)

# add film 'The Wolf of wall Street'
films.append('The Wolf of Wall Street')
print(films)

# length of list films
print(len(films))

# find a film by it's index
print(films[3])

# duration of films
duration = [180, 101, 90, 100, 96, 170]
print(duration)

# good or bad film
good_film = True
print('We will watch')

good_film = False
print('We will not watch')

# create dictionary
duration_films = {'Avatar': 180, 'Dumb and Dumber': 101, 'Pretty Woman': 90, 'The Grinch': 100, 'Man in black': 96, 'The Wolf of Wall Street': 170}


# add new values
duration_films['Harry Potter'] = 160

print(duration_films)

# verify duration
print(duration_films['Pretty Woman'])

# update values
duration_films['Dumb and Dumber'] = 95
print(duration_films)

# delete values
duration_films.pop('Man in black')
duration_films.get('Man in black', 'Will not play anymore')
print(duration_films)